spec/forms/curation_concerns/basic_geo_metadata_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe CurationConcerns::BasicGeoMetadataForm do
+  before do
+    class TestModel < ActiveFedora::Base
+      property :bounding_box,
+               predicate: ::RDF::URI.new('http://www.georss.org/georss/box'),
```

Comment:
Should there be a separate constant used in place of the URI string `http://www.georss.org/georss/box`?

I would obviously defer to @drh-stanford, @eliotjordan, or @yulgit1, but the trend from other issues appears to have been towards abstracting these within the Vocab Module (please see #54).